moveit_ros/move_group/package.xml

```diff
@@ -32,6 +32,10 @@
   <run_depend>pluginlib</run_depend>
   <run_depend>std_srvs</run_depend>
 
+  <test_depend>rostest</test_depend>
+  <test_depend>rosunit</test_depend>
```

Comment:
Are you sure `rosunit` is required? Looks like you are using only standard python `unittest` and `rostest`.

Reply:
Hi Robert, you are right. `rosunit` is needless. I removed it.